docs/building-on-etherlink/tokens.mdx

```diff
@@ -31,35 +31,23 @@ Click the address to go to the block explorer page for the token:
   <td><InlineCopy code="0xc9B53AB2679f573e480d01e0f49e2B5CFB7a3EAb" href="https://explorer.etherlink.com/address/0xc9B53AB2679f573e480d01e0f49e2B5CFB7a3EAb">0xc9B...3EAb</InlineCopy></td>
   <td><InlineCopy code="0xB1Ea698633d57705e93b0E40c1077d46CD6A51d8" href="https://testnet.explorer.etherlink.com/address/0xB1Ea698633d57705e93b0E40c1077d46CD6A51d8">0xB1E...51d8</InlineCopy></td>
 </tr>
-<tr>
-  <td>Etherlink USD</td>
-  <td>eUSD</td>
-  <td>Coming soon</td>
-  <td><InlineCopy code="0x1A71f491fb0Ef77F13F8f6d2a927dd4C969ECe4f" href="https://testnet.explorer.etherlink.com/address/0x1A71f491fb0Ef77F13F8f6d2a927dd4C969ECe4f">0x1A7...Ce4f</InlineCopy></td>
-</tr>
 <tr>
   <td>Tether USD</td>
   <td>USDT</td>
   <td><InlineCopy code="0x2C03058C8AFC06713be23e58D2febC8337dbfE6A" href="https://explorer.etherlink.com/address/0x2C03058C8AFC06713be23e58D2febC8337dbfE6A">0x2C0...fE6a</InlineCopy></td>
-  <td><InlineCopy code="0xD21B917D2f4a4a8E3D12892160BFFd8f4cd72d4F" href="https://testnet.explorer.etherlink.com/address/0xD21B917D2f4a4a8E3D12892160BFFd8f4cd72d4F">0xD21...2d4F</InlineCopy></td>
+  <td><InlineCopy code="0xf7f007dc8Cb507e25e8b7dbDa600c07FdCF9A75B" href="https://testnet.explorer.etherlink.com/address/0xf7f007dc8Cb507e25e8b7dbDa600c07FdCF9A75B">0xD21...2d4F</InlineCopy></td>
```

Comment:
This is not the good address for the data inside the InlineCopy "0xD21...2d4F". Same for the rest of the tokens.

Reply:
> This is not the good address for the data inside the InlineCopy "0xD21...2d4F". Same for the rest of the tokens.

Whoops, I didn't update the `0x###...####` blocks. How does it look now?

Reply:
Perfect.